Plots/MapProjections/NCL_coast_1.py

```diff
@@ -4,14 +4,15 @@
 This script illustrates the following concepts:
    Concepts illustrated:
```

Comment:
This indentation is inconsistent with other files in the gallery and it affects how the doc page looks.